tests/basic/config_mappings.py

```diff
@@ -0,0 +1,29 @@
+from functools import partial as p
+
+from tests.helpers.assertions import has_datapoint
+from tests.helpers.util import run_agent, wait_for
+from tests.monitors.kafka.kafka_test import run_kafka
+
+CONFIG = """
+observers:
+  - type: docker
+monitors:
+  - type: collectd/kafka
+    discoveryRule: container_name == "kafka-discovery" && port == 7099
+    configEndpointMappings:
+      extraDimensions: 'Get(container_labels, "com.signalfx.extraDimensions")'
+      clusterName: 'Get(container_labels, "com.signalfx.cluster")'
```

Comment:
any harm in testing contains?

Reply:
Nope, but it isn't really relevant to the config mapping.  I should probably have a test for boolean values though.